specifyweb/frontend/js_src/lib/hooks/useAsyncState.tsx

```diff
@@ -73,30 +73,34 @@ export function useMultipleAsyncState<RESPONSE extends Record<any, unknown>>(
     | undefined,
   loadingScreen: boolean
 ): GetOrSet<Partial<RESPONSE>> {
-  const [state, setState] = React.useState<Partial<RESPONSE>>({});
   const loading = React.useContext(LoadingContext);
-
+  const [state, setState] = React.useState<Partial<RESPONSE>>({});
```

Comment:
You were going to add `const stateRef = React.useState(state);` and use it to get old value for state instead of `setState((oldState)=>...)`